ruby-bionic/2.7.x/hooks/post_push

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -ex
+
+if [ $DOCKER_TAG != "latest" ]; then
+    MINOR_VERSION=$(echo $DOCKER_TAG | cut -d "." -f 1-2)
+
+    RUBY_VERSION=$(docker run --entrypoint ruby-version.rb $IMAGE_NAME)
+
+    docker tag $IMAGE_NAME $DOCKER_REPO:$RUBY_VERSION
+    docker push $DOCKER_REPO:$RUBY_VERSION
+else 
+    echo "Not retagging latest."
```

Comment:
Can we maybe make this a touch more verbose?  So it makes sense to the poor souls who are debugging this in 5 years. :joy:   And maybe we can also have a corresponding echo when it does pick up a tag like "tagging image with ruby version blah blah".

Reply:
Yeah, that's probably a good idea. I was thinking since this gets run by docker hub so no one will ever see the output anyway.

But if someone looks at this repo in a year or so, they might be wondering what the heck a hook/post_push file does